### ISSUE TITLE
Materials, Items and Professions

### ISSUE THREAD

**Issue by mlloy** [Tirocupidus/TheExiledRPOverhaul#77]
I saw a comment on discord that resonated strongly with me, and I sort of feel like the population of the server to sort of justify how much interdependence there is.  Here is the comment:

"I'm all for "interdependency" with the crafts, but I'd prefer it be of the nature of "I need to build a house, so I need to trade with a builder", or "I need a good steel sword - better find a good smith" than "I need to craft something I should know how to make given my crafting training - better find four other crafters just to make basic tools (which I should have been trained how to make) and supplies (ditto) before I simply hit the "make X items" button"

I honestly think we should limit the amount of points people get to 6 or 8, but make the profession trees more encompassing.  "Ingredient" materials should be open to the paths that need them, but the final product should be limited to the profession-choice.  As an example, if someone has chosen to become a blacksmith, they should have access to Steel-fire by virtue of their profession.  Builders should be able to craft "builder-oriented" materials like iron reinforcements or steel reinforcements, and also have access to the steel-fire to do so - same as woodsman with metal arrows.  These raw ingredients should not be restricted by profession.  I can count a number of times my clan's builder has been online, with very few people on the server, and not able to progress because he a. cannot find anyone to trade with and b. our alchemists aren't online to make Steel-fire.  

With no centralized or travelling merchants, this becomes even more of an issue because we cannot just go and purchase the materials we need either. 

TL;DR I think inter-dependency is great but we haven't found the balance between this and play-ability for the server esp. when considering population sizes.  I think the answer is limiting profession choice, but actually allowing those professions to fully craft the items available for them.  (pps. Smiths should totally be able to make steel-fire).

**Comment by gjpeebles** [comment]
We have moved this direction, closing.
